pages/submit_changes.php

```diff
@@ -25,38 +25,56 @@
 
     $city = filter_var(trim($_POST['city']),FILTER_SANITIZE_STRING);
 
-
     if(mb_strlen($login) <= 5 || mb_strlen($login) >= 25){
         echo "<h1>Логин слишком маленький</h1>";
         exit();
     }
-    if(mb_strlen($description) <= 8 || mb_strlen($description) >= 30){
-        echo "<h1>Описание слишком маленькое</h1>";
-        exit();
+
+    if(mb_strlen($description) != 0){
+        if(mb_strlen($description) <= 8 || mb_strlen($description) >= 30){
+            echo "<h1>Описание слишком маленькое или слишком большое</h1>";
+            exit();
+        }
     }
-    if(mb_strlen($email) <= 7 || mb_strlen($email) >= 50){
-        echo "<h1>Email не существует</h1>";
-        exit();
+    
+    if(mb_strlen($email) != 0){
+        if(mb_strlen($email) <= 7 || mb_strlen($email) >= 50){
+            echo "<h1>Email не существует</h1>";
+            exit();
+        }
     }
-    if(mb_strlen($first_name) <= 2 || mb_strlen($first_name) >= 30){
-        echo "<h1>Имя не существует</h1>";
-        exit();
+
+    if(mb_strlen($first_name) != 0){
+        if(mb_strlen($first_name) <= 2 || mb_strlen($first_name) >= 30){
+            echo "<h1>Имя не существует</h1>";
```

Comment:
Change message so that it corresponds to validation logic. Specify min and max boundaries in error message